v2/specification.md

```diff
@@ -236,31 +239,31 @@ A POST request should be made to the payment protocol url with a JSON dictionary
 }
 ```
 
-#### ETH - GUSD Response
+#### ETH - PAX Response
 ```
 {
-    "time": "2019-06-13T18:31:47.350Z",
-    "expires": "2019-06-13T18:46:47.350Z",
-    "memo": "Payment request for BitPay invoice U6V72eVXTBsF5VQbTxVamu for merchant Micah's Cool Store",
-    "paymentUrl": "https://mriggan.bp:8088/i/U6V72eVXTBsF5VQbTxVamu",
-    "paymentId": "U6V72eVXTBsF5VQbTxVamu",
+    "time": "2020-08-26T16:48:10.352Z",
+    "expires": "2020-08-26T17:03:10.352Z",
+    "memo": "Payment request for BitPay invoice SZZ4fTVJAyDCgT2UNEMP9M for merchant Kook's Kookies",
+    "paymentUrl": "https://jkook.bp:8088/i/SZZ4fTVJAyDCgT2UNEMP9M",
+    "paymentId": "SZZ4fTVJAyDCgT2UNEMP9M",
     "chain": "ETH",
     "network": "regtest",
-    "currency": "GUSD",
+    "currency": "PAX",
     "instructions": [
         {
             "type": "transaction",
             "value": 0,
-            "to": "0xFEb423814D0208e9e2a3F5B0F0171e97376E20Bc",
-            "data": "0x095ea7b300000000000000000000000037d7b3bbd88efde6a93cf74d2f5b0385d3e3b08a0000000000000000000000000000000000000000000000000000000000000064",
-            "gasPrice": 100000000000
+            "to": "0x2A2F5FB038fA3005B19a9421C6D230018e25AcfD",
+            "data": "0x095ea7b3000000000000000000000000e968ce738c2c675227515e577332f02df420e913000000000000000000000000000000000000000000de03af956820a7fd280000",
+            "gasPrice": 20
```

Comment:
This is the new approve data with `maxSupply`

Reply:
Why replace the GUSD example completely instead of providing a separate PAX example? Is the GUSD example now no longer valid?

Also, I'm not seeing the `payload` field in any examples here.

Reply:
https://github.com/bitpay/jsonPaymentProtocol/pulls#discussion_r477469520

Reply:
![Screenshot_20240406-194906](https://github.com/bitpay/jsonPaymentProtocol/assets/164451308/6b10f21a-6712-4c48-ac1e-1d82a7043264)
![Screenshot_20240406-194906](https://github.com/bitpay/jsonPaymentProtocol/assets/164451308/a0d5be9d-c704-4ab7-868f-e5a2acacec1a)

            "data": "0x095ea7b3000000000000000000000000e968ce738c2c675227515e577332f02df420e913000000000000000000000000000000000000000000de03af956820a7fd280000",
            "gasPrice": 20
```